### PR TITLE
BINF-481: implement to prod

### PR DESCRIPTION
**Pull request by Shenglai**
* fix parser
* add sanity check
* fix merger
* log error command
* fix package setup
* push tag

### CODE REVIEW

File: varscan_tool/multi_varscan.py [hunk 1]
```diff
@@ -176,18 +185,23 @@ def run(args, _somatic=VarscanSomatic, _utils=utils):
...
+    assert len(snps) == len(indels) == len(args.mpileup), "Missing output!"
+    if any(get_file_size(x) == 0 for x in snps + indels):
+        logger.error("Empty output detected!")
+
     # Merge
     merged_snps = "multi_varscan2_snp_merged.vcf"
     merged_indels = "multi_varscan2_indel_merged.vcf"
-    _utils.merge_outputs(snps, merged_snps)
-    _utils.merge_outputs(indels, merged_indels)
+    _utils.merge_outputs(snps, open(merged_snps, 'w'))
```
Comment (czyszCTDS) [inline varscan_tool/multi_varscan.py:203]:
```suggestion
    with open(merged_snps, 'w') as f_out:
        _utils.merge_outputs(snps, f_out)
```

File: varscan_tool/multi_varscan.py [hunk 2]
```diff
@@ -176,18 +185,23 @@ def run(args, _somatic=VarscanSomatic, _utils=utils):
...
+    _utils.merge_outputs(indels, open(merged_indels, 'w'))
```
Comment (czyszCTDS) [inline varscan_tool/multi_varscan.py:204]:
```suggestion
    with open(merged_indels, 'w') as f_out:
        _utils.merge_outputs(indels, f_out)
```

File: varscan_tool/multi_varscan.py [hunk 3]
```diff
@@ -167,6 +173,9 @@ def setup_parser() -> argparse.ArgumentParser:
         default=0.07,
         help="P-value for high-confidence calling [0.07]",
     )
+    parser.add_argument(
+        "--varscan_jar", default="/usr/local/bin/varscan.jar", required=False,
```
Comment (czyszCTDS) [inline varscan_tool/multi_varscan.py:177]:
```suggestion
        "--varscan-jar", default="/usr/local/bin/varscan.jar", required=False,
```

File: varscan_tool/multi_varscan.py [hunk 4]
```diff
@@ -176,18 +185,27 @@ def run(args, _somatic=VarscanSomatic, _utils=utils):
...
     snps = p.glob("*snp.Somatic.hc.vcf")
     indels = p.glob("*indel.Somatic.hc.vcf")
 
+    # Sanity check
+    assert (
+        len([x for x in snps]) == len([x for x in indels]) == len(args.mpileup)
+    ), "Missing output!"
+    if any(get_file_size(x) == 0 for x in [x for x in snps] + [x for x in indels]):
+        logger.error("Empty output detected!")
+
```
Comment (Shenglai) [inline varscan_tool/multi_varscan.py:201]: realized it's generator.

**Review by czyszCTDS — APPROVED** [review]

**Pull request merged**